execution_engine/src/core/runtime_context/mod.rs

```diff
@@ -566,8 +570,8 @@ where
     }
 
     /// Write an era info instance to the global state.
-    pub fn write_era_info(&mut self, key: Key, value: EraInfo) {
-        if let Key::EraInfo(_) = key {
+    pub fn write_era_summary(&mut self, key: Key, value: EraInfo) {
+        if let Key::EraSummary = key {
```

Comment:
i don't think we can get away w/ changing the name of this public function in 1.x

instead, consider making a new function with the new behavior and forward the old function to it, mark this fn as deprecated, and todo! remove it in 2.0

Reply:
Good call, I renamed it back (although I think we would be fine, as we don't reimport RuntimeContext as pub)